README.md

```diff
@@ -5,7 +5,18 @@ Exercism Exercises in Elm
 
 ## Setup
 
-Detailed installation instructions can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+- Detailed installation instructions for elm can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+
+- Install `elm-test` for the command line with `npm install elm-test`.
+
+### Running Tests in an Exercise Directory
+Inside an exercise directory, Elm must be configured manually.
+
+- Install `laszlopandy/elm-console` and `deadfoxygrandpa/elm-test`.
+  - `elm-package install -y laszlopandy/elm-console`
+  - `elm-package install -y deadfoxygrandpa/elm-test`
+- You should then be able to run tests with `elm-test ExerciseModuleNameTest.elm`
+
```

Comment:
As another bonus (and I just double checked this), the first time you run `elm-test` from the root it will ask for permission to install all of the elm packages specified in elm-package.json.